engine/table/src/main/java/io/deephaven/engine/table/impl/select/MatchFilter.java

```diff
@@ -167,12 +138,11 @@ private MatchFilter(
 
     public MatchFilter renameFilter(String newName) {
         if (strValues == null) {
-            return new MatchFilter(expression, parserConfiguration, failoverFilter, getMatchType(), newName, values);
+            return new MatchFilter(getMatchType(), newName, values);
```

Comment:
This is effectively asserting that there can be no failover if we have values. I think it's accurate. Might need a comment.